docs/src/folderstruct.md

```diff
@@ -0,0 +1,55 @@
+
+The folder /docs contains:
+- a `/src` folder: It is recommended to put your markdown pages inside this folder. Each markdown file (.md extension text file) is a page accessible through the menu of the documentation. For example, `docs/src/Contributing.md` contains the text you can read on the documentation "contribution guide" menu. The path to this .md file and the name of the menu is set in the `docs/make.jl` file. 
+- a `make.jl` file: This Julia file contains your documentation website structure. Running this file will build your website pages, you can run it locally, but it is commonly built remotely via .github/workflows/docs.yml to generate the github static page hosted on the [gh-pages branch](https://github.com/CliMA/ClimaLSM.jl/tree/gh-pages). 
+
+Note: the documentation can have sub-menu. For example, `APIs` have seb-menu `ClimaLSM` which has many sub-menu... This structure is built in our current framework via a file `docs/list_of_apis.jl` in that example, which is then included in `docs/make.jl`.   
+
+## /experiments folder
+
+The `experiments` folder contains scripts to run ClimaLSM models. It contains a folder for `integrated` models and a folder for `standalone` models. It is meant to provide users with simple examples of ClimaLSM runs. The files contains meteorological inputs (such as precipitation), values for every parameters, and the domains and timestepper are specified. 
+
+For example, `/experiments/LSM/ozark/` contains:
+- ozark_domains.jl: Describes the soil domain (depth, number of layer), and the canopy (number and height of stems and leaves).
+- ozark_met_drivers_FLUXNET.jl: This files load meteorological input data from the ozark FLUXNET file, and does additional things such as spline interpolation of these drivers.
+- ozark_parameters.jl: In this file, parameters values are defined, for example the soil porosity is given a value of 0.55 m3 m-3. 
+- ozark_simulation.jl: In this file, initial and final time are set, as well as time resolution and time stepper algorithm. 
+- ozark.jl: running this script will include all the above scripts, and run ClimaLSM for the single-site ozark. It will produce output in a text file as well as some figures comparing data and simulation.
+
+## /src folder
+
+The `/src` folder contains the source code of `ClimaLSM` models. It contains 3 folders:
+- SharedUtilities: This is a core folder that defines functions to create `ClimaLSM` models, and many utility functions to be called across the repository. For example, `SharedUtilities/models.jl` defines and export the function `make_update_aux` which will be used to store auxiliary variables in a nested variable `p`, `SharedUtilities/boundary_conditions.jl` defines functions for setting boundary condition for ODE domains, etc.
```

Comment:
"which will be used to create a function which updates the auxiliary parameters, stored in the vector `p`"